lib/machine-loader.js

```diff
@@ -34,15 +36,40 @@ function getConfig (defaultConfig) {
 }
 
 function getMachineNames (config) {
+  const fullyFunctionalStatus = {label: 'Fully functional', type: 'success'}
+  const unresponsiveStatus = {label: 'Unresponsive', type: 'error'}
+  const stuckStatus = {label: 'Stuck', type: 'error'}
+  
   return Promise.all([getMachines(), getConfig(config)])
-    .then(([machines, config]) => {
+    .then(([machines, config]) => Promise.all(
+      [machines, notifier.checkPings(machines), dbm.machineEvents(), config]
+    ))
+    .then(([machines, pings, events, config]) => {
+      const getPingStatus = (ping) => {
+        if (!ping) return fullyFunctionalStatus
+
+        if (ping.age) return unresponsiveStatus
+        
+        return fullyFunctionalStatus
+      }
+      
+      const getStuckStatus = (stuck) => {
+        if (!stuck || !stuck.age) return undefined
+      
+        return stuckStatus
+      }
```

Comment:
Mentioned this on another review, but we don't really want to show both "Fully functional" and "Stuck". If it's stuck it can't be fully functional.